helper/testhelpers/testhelpers.go

```diff
@@ -565,6 +574,35 @@ func VerifyRaftConfiguration(core *vault.TestClusterCore, numCores int) error {
 	return nil
 }
 
+// AwaitLeader waits for one of the cluster's nodes to become leader.
+func AwaitLeader(t testing.T, cluster *vault.TestCluster) (int, error) {
+
+	timeout := time.Now().Add(30 * time.Second)
+	for {
+		if time.Now().After(timeout) {
```

Comment:
nit: you could also just use a `select`  here with a ticker and a timeout

Reply:
Yes, that's true.  I'll leave it as it is for now but its a better motif to use going forwards